packages/css/src/popover.css

```diff
@@ -4,13 +4,13 @@
   --dsc-popover-color: var(--ds-color-text-default);
   --dsc-popover-arrow-size: var(--ds-size-2);
   --dsc-popover-border-radius: var(--ds-border-radius-md);
-  --dsc-popover-border: 1px solid var(--dsc-popover-border-color);
   --dsc-popover-max-width: 300px;
   --dsc-popover-padding: var(--ds-size-3) var(--ds-size-4);
 
   background: var(--dsc-popover-background);
   border-radius: var(--dsc-popover-border-radius);
-  border: var(--dsc-popover-border);
+  border: 1px solid;
```

Comment:
Skal jeg ikke få endre på border-bredde? :)

Reply:
Yes stemmet det. Det er kun farge på border som blir endra av css variabel. 
Skal du endre width og stil, så endrer du css prop `border:`, slik som du ville ha endra `display:`.
Skal me ha eigen css var for noke som me ikkje endra, så vil det potensielt bli forventa at me skal gjere det for alt.

Reply:
ser poenget 🤝

Reply:
ser eg heller ikkje har nevnt ein stort poeng med at eg gjorde det :P :
Om me har ein variabel som endrer for heile `border:`, så må du ta stilling til både stil, width og farge. Kanskje me bruker ein fargevariabel, då må du finne den - så å ha eigen for farge tar bort pain-points med at du må hugse å legge på rette greier.